core/src/com/cloud/agent/api/StartupRoutingCommand.java

```diff
@@ -35,7 +35,7 @@
     long memory;
     long dom0MinMemory;
     boolean poolSync;
-
+    private boolean _supportsClonedVolumes;
```

Comment:
what about starting to use variables without "_"?
I know ACS is full of variables starting with "_", but I believe we should start going toward a better pattern.